doc/authen_pause.schema.txt

```diff
@@ -56,6 +56,9 @@ CREATE TABLE usertable (
   `changed` int(11) DEFAULT NULL,
   changedby char(10) DEFAULT NULL,
   lastvisit datetime DEFAULT NULL,
+  mfa tinyint(1) DEFAULT 0,
+  mfa_secret32 varchar(16) DEFAULT NULL,
+  mfa_recovery_codes text DEFAULT NULL,
```

Comment:
I feel like I'm probably going to ask a *lot* of questions.  I hope it's useful and not just annoying.

1. Does `mfa` mean "MFA is enabled"?  If so, is there any value to having that *and* `mfa_secret32`?  I would expect that you can't temporarily turn off MFA and turn it back on with the same secret, so no extra field is needed.

Reply:
It would be good to store `mfa_last_verified_at` with the time window in which it was verified.  If a OTP can be used more than once, it isn't a OTP!  Better practice is something like:
* TOTP tokens change every 30s
* we accept the TOTP token for now, 30s ago, and 30s from now
* we store the time seed of any verified TOTP
* we never accept a TOTP with a seed less than or equal to the last one

Otherwise, the token used at 12:00:00 has 89 seconds to be exfiltrated and used again.

Reply:
> Does mfa mean "MFA is enabled"? If so, is there any value to having that and mfa_secret32

Maybe we don't need `mfa` column anymore.

Reply:
Removed mfa column with https://github.com/andk/pause/pull/455/commits/e1549a564d4e9ccd3aac35c2f730c14e71f77a7b